### DISCUSSION TITLE
Duda con muestra de input output parte 1

### DISCUSSION THREAD

**Discussion by MaxAl100** [IIC2133-PUC/2024-1#63] [Tarea 1] [answered]
Me parece que hay un error (o entiendo mal el ejercicio) en la penúltima línea de output de la parte 1, ya que dice "User 1 compra a 1320" pero es el usuario 7 que hace la orden de compra, no? O estoy entendiendo algo mal?

**Comment by benjavicente** [comment, marked as answer]
Hola!

El output de ejemplo de esa parte estaba incorrecto respecto a lo que se menciona en el problema. Se subió ahora el arreglo al enunciado.

También, vamos a revisar los outputs subidos en el drive por si tienen el mismo error.
